docs/resources/lakehouse.md

```diff
@@ -23,8 +132,71 @@ Supports a multitude of computing environments for cloud-native storage. Users c
 
 Incorporates a hosted implementation of Apache Iceberg REST catalog, facilitating interaction with DataOS Lakehouse through Iceberg REST catalog API.
 
+## Structure of a Lakehouse manifest
+
+```yaml hl_lines="1-9 11-15 17-26 28-31 33-36"
+
+# Resource meta section (1)
+name: <lakehouse-resource-name> 
+version: v1alpha
+layer: user
+type: lakehouse        
+tags:
+  - <tag1>
+  - <tag2>
+description: <lakehouse-resource-description>
+
+# Lakehouse-specific section (2)
+lakehouse:
+  type: iceberg
+  compute: <compute-name>
+  iceberg:
+
+    # Storage section (3)
+    storage:
+      type: <storage-type> 
+      <storage-type>:
+        format: iceberg
+        icebergCatalogType: <iceberg-catalog-type>
+        metastoreType: <metastore-type>
+        metastoreUrl: <url>
+        relativePath: <relative-path>
+        # ...other Storage-specific attributes
+
+    # Metastore section (4)
+    metastore:
+      type: <metastore-type>
+      # ...other Metastore-specific attributes
+
+    # Query engine section (5)
+    queryEngine:
+      type: <query-engine-type>
+      # ...other Query-engine-specific attributes
+```
+
+1.  **Resource meta section** within a manifest file comprises metadata attributes universally applicable to all [Resource-types](./types_of_dataos_resources.md). To learn more about how to configure attributes within this section, refer to the link: [Attributes of Resource meta section](./resource_attributes.md).
+
+2.  **Lakehouse-specific section** within a manifest file comprises attributes
+    specific to the Lakehouse Resource. This section is further subdivided into: Storage, Metastore, and Query Engine section. To learn more about how to configure attributes of Lakehouse-specific section, refer the link: [Attributes of Lakehouse-specific section](./lakehouse/yaml_configuration_attributes.md).
+
+3.  :man_raising_hand: I'm a code annotation! I can contain `code`, __formatted
+    text__, images, ... basically anything that can be written in Markdown.
+
+4.  :man_raising_hand: I'm a code annotation! I can contain `code`, __formatted
+    text__, images, ... basically anything that can be written in Markdown.
+
+5.  :man_raising_hand: I'm a code annotation! I can contain `code`, __formatted
+    text__, images, ... basically anything that can be written in Markdown.
+
+
 ## How to create a Lakehouse?
 
+### **Prerequisites**
+
+#### **Create Read and Read-write Instance Secrets**
```

Comment:
We should probably get rid of the entire instance secret part from here. Let's talk about it in person.
Also, the YAML of instance secret should come before the tabs showcasing how these secrets are filled for different storage accounts.